src/components/LayerUI.tsx

```diff
@@ -65,6 +65,7 @@ interface LayerUIProps {
   renderCustomFooter?: (isMobile: boolean) => JSX.Element;
   viewModeEnabled: boolean;
   libraryReturnUrl: ExcalidrawProps["libraryReturnUrl"];
+  showExport: boolean;
```

Comment:
we should pass the `UIOptions` props here instead coz later on `selectedShapeActions` will also be requiring this so individual prop for each is not a good idea